lib/scottkit/game.rb

```diff
@@ -106,8 +110,24 @@ def initialize(options)
       @options = options
       @rooms, @items, @actions, @nouns, @verbs, @messages =
         [], [], [], [], [], []
+      @output = options[:output] || $stdout
+    end
+
+    def puts *args
```

Comment:
This should probably be `#:nodoc:` right?

Reply:
I don't know about `:nodoc` -- I've abandoned the idea of using any of the inline documentation things, as I don't think anyone ever actually uses the resulting documents. See #19.

Reply:
I'm not sure about overriding `print` in `game.rb`. Isn't the more honest approach you used in the previous commit also suitable here?